File: Examples/DotNet/CSharp/Charts/GetChart.cs
using System;
using Com.Aspose.Cells.Api;
using Com.Aspose.Cells.Model;
using Com.Aspose.Storage.Api;

namespace Charts
{
    class GetChart
    {
        public static void Run()
        {
            // ExStart:1
            CellsApi cellsApi = new CellsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "Sample_Test_Book.xls";
            String sheetName = "Sheet5";
            int chartNumber = 0;
            String storage = "";
            String folder = "";

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.Cells Cloud SDK API to get chart from worksheet
                ResponseMessage apiResponse = cellsApi.GetWorksheetChart(fileName, sheetName, chartNumber, storage, folder);

                if (apiResponse != null)
                {
                    Console.WriteLine("Get Chart from a Worksheet, done!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/Android/app/src/main/java/com/aspose/cells/cloud/examples/chart/GetChart.java
package com.aspose.cells.cloud.examples.chart;


import android.content.Context;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.R;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;
import java.io.File;

import java.io.IOException;


public class GetChart {

	public static void execute(Context context) throws IOException {
		//ExStart: get-chart-from-worksheet
		try {
			// Instantiate Aspose Storage API SDK
			StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

			// Instantiate Aspose Words API SDK
			CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
			String input = "Sample1.xlsx";


			File inputFile = Utils.stream2file("sample1","xlsx", context.getResources().openRawResource(R.raw.sample1));

			String sheet = "Sheet1";
			int chart = 0;

			storageApi.PutCreate(input, null, Utils.STORAGE, inputFile);

			com.aspose.cells.model.ChartResponse r = cellsApi.GetWorksheetChart(input, sheet, chart, Utils.STORAGE,
					Utils.FOLDER);

			System.out.println("Name: " + r.getChart().getName());
			System.out.println("Type: " + r.getChart().getType());

		}

		catch (Exception e) {
			e.printStackTrace();
		}
		//ExEnd: get-chart-from-worksheet
	}
}